applications/callflow/test/cf_ring_group_tests.erl

```diff
@@ -28,3 +28,79 @@ weighted_random_sort_test_() ->
       || X <- Endpoints
      ]
     ].
+
+%% HELP-260010392
+help_10392_test_() ->
+    UserId = <<"help-10392-user-id">>,
+    DevId = <<"help-10392-device-id">>,
+    GroupId = <<"help-10392-group-id">>,
+
+    meck:new('kz_datamgr', ['passthrough']),
```

Comment:
You are the only one using meck afaict. I would prefer tests to use fixturedb as that is where testing needing database access has gone in the project. There are a number of tests showing examples (like [webhooks](https://github.com/2600hz/kazoo/blob/master/applications/webhooks/test/webhooks_disabler_tests.erl)) of how make this happen.